.github/workflows/build.yml

```diff
@@ -176,6 +176,7 @@ jobs:
           cargo check --target="${target}" --features="portable-atomic-critical-section"
           cargo check --target="${target}" --features="serde"
           cargo check --target="${target}" --features="ufmt"
+          cargo check --target="${target}" --features="embedded-io"
```

Comment:
Please add the embedded-io feature to the tests and to the miri test runs too

Reply:
How do you want this done? None of the other features appears to be tested in this manner, so I'm not sure what you expect.

Reply:
Add the embedded-io feature line 51 and 87 of the `.github/workflows/build.yml` file.